src/structures/aligned_segment.cpp

```diff
@@ -0,0 +1,25 @@
+#include "structures/aligned_segment.hpp"
+
+#include <seqan3/alphabet/cigar/cigar.hpp>
+
+#include "structures/breakend.hpp"          // for strand
+
+/*! \brief Read segment aligned to the reference genome (part of chimeric/split-aligned read). Contains information
+ *        parsed from the SA tag of an alignment in the SAM/BAM file.
+ *
+ * \param orientation       mapping orientation (reverse or forward strand)
+ * \param ref_name          reference/chromosome name
+ * \param pos               start position of the alignment
+ * \param mapq              mapping quality
+ * \param cig               cigar string of the alignment
+ */
+struct AlignedSegment;
```

Comment:
This doesn't belong in the cpp file. Documentation goes to the header file and the declaration of `AlignedSegment` is also there and can be omitted here.